integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/car/deployment/test/CARBON16113DeployArtifactsBeforeTransportStarsTestCase.java

```diff
@@ -42,45 +36,28 @@
     private final String TRANSPORT_MESSAGE = "Pass-through HTTP Listener started on ";
     private final String CAPP_MESSAGE = "Deploying Carbon Application : car-deployment-before-tranaport-start-test_1.0.0.car";
     private ServerConfigurationManager serverConfigurationManager;
-    private LogViewerClient logViewerClient;
+    private CarbonLogReader carbonLogReader;
 
     @BeforeClass(alwaysRun = true)
     private void initialize() throws Exception {
         super.init();
-        serverConfigurationManager = new ServerConfigurationManager(
-                new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN));
+        carbonLogReader = new CarbonLogReader();
+        serverConfigurationManager = new ServerConfigurationManager(new AutomationContext());
+        carbonLogReader.start();
+        serverConfigurationManager.restartMicroIntegrator();
+        carbonLogReader.stop();
```

Comment:
Are we starting and stoping log reader to assert startup log or missed to remove?

Reply:
we can do this in test case rather than in init

Reply:
In this testcase we get all the logs related to server startup, and therefore it is kind of a lengthy log string. if we do it later then it can be a large string and it takes more memory to evaluate. That's why in here, stop the log reader right after all the logs are retrieved.